napari_hub_cli/checklist/metadata.py

```diff
@@ -248,10 +249,25 @@ def display_checklist(analysis_result):
 
     # Display summary result
     previous_title = None
+    section_statuses = defaultdict(bool)
+    section_check_mark = "PASS \N{CHECK MARK}"
+    section_x_mark = "FAIL \N{BALLOT X}"
+
+    for feature in analysis_result.features:
+        if feature.meta.section:
+            section_title = feature.meta.section.title
+            section_statuses[section_title] |= feature.found == False
+    
+    last_section_status = dict(section_statuses)
+
     for feature in analysis_result.features:
         if feature.meta.section and feature.meta.section.title != previous_title:
-            console.print()
-            console.print(feature.meta.section.title, style="bold underline white")
+            if  last_section_status[feature.meta.section.title] == False:
```

Comment:
@SimaoBolota-MetaCell here, you can "invert" the two ifs and then simplify them from 
```python
if last_section_status[feature.meta.section.title] == True:
    ...
if last_section_status[feature.meta.section.title] == False:
    ...
```

to 

```python
if last_section_status[feature.meta.section.title]:
    ...
else:
    ...
```

Reply:
Thank you!